src/V3DupFinder.cpp

```diff
@@ -0,0 +1,99 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: Hashed common code into functions
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2003-2021 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+
+#include "config_build.h"
+#include "verilatedos.h"
+
+#include "V3Global.h"
+#include "V3DupFinder.h"
+#include "V3Ast.h"
+#include "V3File.h"
+
+#include <algorithm>
+#include <iomanip>
+#include <map>
+#include <memory>
+
+//######################################################################
+// V3DupFinder class functions
+
+V3DupFinder::V3DupFinder() {}
```

Comment:
Move to header, then can use "V3DupFinder() = default;"

Reply:
Clang says:
Explicitly defaulted default constructor is implicitly deleted default constructor of 'V3DupFinder' is implicitly deleted because field 'm_hasher' of const-qualified type 'const V3Hasher' would not be initialized